v2/azure-arm.yaml

```diff
@@ -31,6 +31,8 @@ supportedResourcesReport:
   # resourcePath is a template used for generating a file path for checking whether docs for a resource have been generated
   # specified relative to the directory of outputPath, and supports the same placeholders
   resourcePathTemplate: "{group}/{version}.md"
+  # latestRelease is the version of our current release (resources introduced in newer releases will be shown as unreleased)
+  latestRelease: v2.0.0
```

Comment:
The version is injected into each binary as part of the build process with this cmd from taskfile:
`-ldflags "-X {{.PACKAGE}}/internal/version.BuildVersion={{.VERSION}}"`

You should be able to read the `BuildVersion` variable, which will be of the form `v2.0.0-rev.72`. If you just cut off the "additional stuff" after the `-`, you'll have the most recent release.

That way you don't need to hard-code this, it can be determined dynamically.

Reply:
Good point! This way we don't have to hardcode the version manually on each release.

Reply:
I tried this out - but the variable is only defined when `generator` is build by the `Taskfile` - so it's not available when running tests (e.g. the tests that the report works, nor if you're running the generator by hand. I'm open to making this change, but there seem to be a number of thorns, so I'll push it out to another PR.

Reply:
Completed in this PR. Uses `version.BuildVersion` if available, falls back to using `git` commands if not.